src/bio/bio_recovery.c

```diff
@@ -203,16 +218,21 @@ bio_bs_state_set(struct bio_blobstore *bbs, enum bio_bs_state new_state)
 		break;
 	default:
 		rc = -DER_INVAL;
-		D_ASSERTF(0, "Invalid bs state: %u\n", new_state);
+		D_ASSERTF(0, "Invalid bs state: %u (%s)\n",
+			  new_state, bio_state_enum_to_str(new_state));
 		break;
 	}
 
 	if (rc) {
-		D_ERROR("BS state transit error! tgt: %d, %u -> %u\n",
-			bbs->bb_owner_xs->bxc_tgt_id, bbs->bb_state, new_state);
+		D_ERROR("BS state transition error! tgt: %d, %s -> %s\n",
```

Comment:
What does BS stand for?   I mean, I have a guess but I don't think it's correct :-)

Reply:
updated in #1469